examples/async_ssl_server.py

```diff
@@ -37,7 +37,7 @@ def alive(self, **kwargs):
 
 class Server:
```

Comment:
I'm wondering if it would be worthwhile to have an alternative api to use the internal KJ TLS as well?

It might be a bit simpler of an API (depending on how much work it is to use). I don't think it's necessary to do as part of this PR unless you think it's easy.

Reply:
See https://github.com/capnproto/pycapnp/issues/311 for my proposal w.r.t. file descriptors. This would solve the TLS issue.